### PR TITLE
Add missing \tcode.

### PR DESCRIPTION
**Pull request by Eelis**
http://eel.is/tcodediffs.pdf (45 pages)

### CODE REVIEW

File: source/compatibility.tex [hunk 1]
```diff
@@ -742,7 +742,7 @@
 \rationale
 When classes become complicated, allowing such a redefinition
 after the type has been used can create confusion for \Cpp
-programmers as to what the meaning of 'I' really is.
+programmers as to what the meaning of '\tcode{I}' really is.
```
Comment (tkoeppe) [inline source/compatibility.tex]: The quotes are bogus. It should say:

    `\tcode{I}'

**Comment by Eelis** [comment]
Rebased, and added the drive-by punctuation fix @tkoeppe suggested.

File: source/compatibility.tex [hunk 2]
```diff
@@ -742,7 +742,7 @@
 \rationale
 When classes become complicated, allowing such a redefinition
 after the type has been used can create confusion for \Cpp
-programmers as to what the meaning of 'I' really is.
+programmers as to what the meaning of `\tcode{I}' really is.
```
Comment (zygoloid) [inline source/compatibility.tex]: Maybe drop the \` and ' too?

Comment (Eelis) [inline source/compatibility.tex]: Fine with me. @tkoeppe would that work for you?

Comment (tkoeppe) [inline source/compatibility.tex]: Yes, no quotes is good, too.

Comment (Eelis) [inline source/compatibility.tex]: Gone

**Comment by jensmaurer** [comment]
Looks good to me.

**Comment by Eelis** [comment]
Rebased.

**Pull request merged**